OpenSim/Tests/Wrapping/testWrapping.cpp

```diff
@@ -617,14 +619,14 @@ void simulateModelWithCables(const string &modelFile, double finalTime)
             else { // next two path points should be a wrap point
                 for (int k = 0; k < wrapSet.getSize(); ++k) {
                     const Vec3& wrapStartPointLoc = wrapSet[k].getPreviousWrap().r1;
-                    if (!wrapStartPointLoc.isInf() && pp->getLocation().isNumericallyEqual(wrapStartPointLoc)) {
+                    if (!wrapStartPointLoc.isInf() && pp->getLocation(si).isNumericallyEqual(wrapStartPointLoc)) {
                         ObstacleInfo* obs = wrapObs[k];
                         obs->isActive = true;
                         // pp and next pp are wrap points
                         Transform X_SB = obs->X_BS.invert();
-                        PathPoint* pp_next = activePathPoints[++i]; // increment to next pp
-                        obs->P_S = X_SB*pp->getLocation();
-                        obs->Q_S = X_SB*pp_next->getLocation();
+                        AbstractPathPoint* pp_next = activePathPoints[++i]; // increment to next pp
+                        obs->P_S = X_SB*pp->getLocation(si);
+                        obs->Q_S = X_SB*pp_next->getLocation(si);
```

Comment:
My sense is that these changes are actually fixing bugs for the case where the active points include MovingPathPoints. Which is great.